### ISSUE TITLE
Travelling through an ME drive with staff of travelling causing crash

### ISSUE THREAD

**Issue by AllenAlien920** [Divine-Journey-2/Divine-Journey-2#1168]
### Modpack version

2.21.0

### Java version

8u421

### RAM Allocated

8GB allocated, 32GB on my PC

### Description

I pointed at an ME drive while holding the Ender IO's staff of travelling, and shift+right click to teleport, and the game window became completely white, then after a few seconds it crashes.

### When did the crash happen?

When I tried teleporting through an ME drive with a staff of travelling.

### Steps to reproduce

1. Create or enter a world
2. Place an ME drive in front of you (should be one or two blocks higher than your current Y level)
3. Hold a staff of travelling and point at the ME drive
4. Shift+right click to teleport through the ME drive
5. The game crashes

### Minecraft crash report

I didn't find the crash report file (though in the latest.log it's said to be generated in the directory), so I just provide a part of latest.log relevant to the crash.
https://paste.ubuntu.com/p/2fd7xJwBW3/

### Additional context

It seems that the cause of the crash is a combination of AE2 and Universal Tweaks, so I created a new version with these two and Ender IO being the only mods, and the crash still happens.

### Your Divine Journey 2 Discord Username

allen_alien920

**Comment by AllenAlien920** [comment]
This crash is probably an incompatibility of the bugfix "Block Overlay" of Universal Tweaks and other mods' transparent blocks, and I have raised an [issue](https://github.com/ACGaming/UniversalTweaks/issues/547) under the mod.
A quick fix for this bug is to disable "Block Overlay" in the config file:
```
general {

    "bugfixes: blocks" {
       ...

        "block overlay" {
            # Fixes x-ray when standing in non-suffocating blocks
            B:"[1] Block Overlay Toggle"=false
       ...
```
Also, I didn't know that there is already a similar issue #1162 here, and I apologize for creating a duplicate one.

**Comment by AllenAlien920** [comment]
The bug is fixed with ae2-uel's update to v0.56.6. Since the mods are updated regularly in the modpack update, I'll just mark this issue as completed without a pull request.

**Comment by WaitingIdly** [comment]
would prefer if it remained opened until properly resolved in DJ2
